src/NavigationState.ts

```diff
@@ -1,13 +1,15 @@
 import {Navigation, CommandName, ModalDismissedEvent, ComponentDidAppearEvent} from 'react-native-navigation';
 
+let activeScreenId: string;
+
 interface State {
   stackCounter: number;
-  activeScreenId?: string;
```

Comment:
we can still export this, to avoid internal breakage with current release cycle. 
we can remove it later on

Reply:
Done